distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -60,14 +60,6 @@ log4j2.logger.ingestLogger.appenderRef.ingestError.ref = ingestError
 log4j2.logger.ingestLogger.appenderRef.syslog.ref = syslog
 log4j2.logger.ingestLogger.appenderRef.osgi-platformLogging.ref = osgi-platformLogging
 
-# org.apache.solr
-log4j2.logger.org_apache_solr.name = org.apache.solr
```

Comment:
❓ I'm not very familiar with logging configuration. How will these changes impact the logging of the solr client related packages?

Reply:
That's actually a good question, I wonder if we should just get rid of the appender ref for solr and keep the syslog one.

Reply:
The solr.log file in the data/log dir always stays at zero bytes. So my guess is that it will not have an effect.